sync/class.jetpack-sync-defaults.php

```diff
@@ -240,6 +240,7 @@ public static function get_constants_whitelist() {
 		'site_icon_url'                    => array( 'Jetpack_Sync_Functions', 'site_icon_url' ),
 		'roles'                            => array( 'Jetpack_Sync_Functions', 'roles' ),
 		'timezone'                         => array( 'Jetpack_Sync_Functions', 'get_timezone' ),
+		'available_jetpack_blocks'         => array( 'Jetpack_Gutenberg', 'get_block_availability' ),
```

Comment:
I propose we rename this to `available_jetpack_editor_extenstions` or `available_jetpack_editor_assets`
cc: @enejb @lezama 

It's not a blocker per-se, but the callable is actually reporting on the availability or blocks AND plugins. If we are going to re-name for accuracy, we should do so before the release.